view/frontend/web/template/payment/rvvup.html

```diff
@@ -26,7 +26,13 @@
             <!--/ko-->
         </div>
 
-        <!-- ko if: (getCode() == 'rvvup_CARD' && rvvup_parameters.settings.card.flow === 'MODAL') -->
+        <!-- ko if: (getCode() != 'rvvup_CARD') -->
```

Comment:
```suggestion
        <!-- ko if: (getCode() != 'rvvup_CARD' || (getCode() == 'rvvup_CARD' && rvvup_parameters.settings.card.flow != 'INLINE')) -->
```

We can do this and avoid the duplicate fragment below.

Reply:
Implemented, thank you!